src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -52,11 +53,23 @@ def load_arguments_eh(self, _):
                    help='A boolean value that indicates whether Managed Identity is enabled.')
         c.argument('disable_local_auth', options_list=['--disable-local-auth'], is_preview=True, arg_type=get_three_state_flag(),
                    help='A boolean value that indicates whether SAS authentication is enabled/disabled for the Event Hubs')
+        c.argument('system_assigned', arg_group='Managed Identity',
+                   arg_type=get_three_state_flag(),
+                   help='Enable System Assigned Identity')
+        c.argument('user_assigned', arg_group='Managed Identity', nargs='+', help='List of User Assigned Identity ids.')
```

Comment:
Better to follow MSI design guideline, see https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md#enable-managed-identity-during-resource-creation

@zhoxing-ms for review